### PR TITLE
[Story of Layout] Improve responsiveness & increase split initial ratio

### PR DESCRIPTION
**Pull request by albertusangga**
- Avoid overflowing when size of the tab is small

## Demo
Kapture 2019-12-04 at 18 10 11

### CODE REVIEW

File: packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart [hunk 1]
```diff
@@ -146,7 +146,7 @@ class _InspectorScreenBodyState extends State<InspectorScreenBody>
         Expanded(
           child: Split(
             axis: splitAxis,
-            initialFirstFraction: splitAxis == Axis.horizontal ? 0.35 : 0.6,
+            initialFirstFraction: 0.35,
```
Comment (jacob314) [inline packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart]: @DaveShuckerow 
What we would really like is for the flex details tab to trigger animating the splitter proportion to a value suitable to display the flex details when the tab is open.

File: packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart [hunk 2]
```diff
@@ -830,29 +836,35 @@ class _StoryOfYourFlexWidgetState extends State<StoryOfYourFlexWidget>
...
+                      ),
                     ),
                   ),
                 ),
                 type: ArrowType.down,
               ),
             ),
-            _buildAxisAlignmentDropdown(Axis.vertical),
+            Flexible(
+              flex: maxHeight <= minHeightToAllowTruncating ? 1 : 0,
```
Comment (jacob314) [inline packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart]: this expression is repeated a few times.  Maybe add a widget that is
Truncateable({bool truncate, Widget child})
as a bonus I can imagine adding some fancier logic for how we truncate later on.

Comment (DaveShuckerow) [reply packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/flex.dart]: +1

**Comment by albertusangga** [comment]
Increased the padding @jacob314 
image

**Comment by albertusangga** [comment]
Can I merge this now? @jacob314  @DaveShuckerow

**Comment by DaveShuckerow** [comment]
> Can I merge this now? @jacob314 @DaveShuckerow

Go ahead!

**Pull request merged**